sematic/api/endpoints/resolutions.py

```diff
@@ -606,11 +622,30 @@ def clean_stale_resolution_endpoint(
 ) -> flask.Response:
     resolution = get_resolution(root_id)
     root_run = get_run(root_id)
+    logger.warning(
```

Comment:
Warning? Not info?

Reply:
I debated. Cleaning stuff usually means there was something that caused the garbage to show up that might be interesting. Will switch to info though.